mne/source_estimate.py

```diff
@@ -1395,7 +1419,10 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
                                       config_opts=config_opts,
                                       subjects_dir=subjects_dir, figure=figure,
                                       views=views, colorbar=colorbar,
-                                      clim=clim)
+                                      clim=clim, cortex=cortex, size=size,
+                                      background=background,
+                                      foreground=foreground,
+                                      time_unit=time_unit)
```

Comment:
Unless we also modify PySurfer, this would have to be done in a call to `Brain.set_time()` anyways...